scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/ClientHandshaker.java

```diff
@@ -471,6 +471,7 @@ private void receivedServerHelloDone(ServerHelloDone message) throws HandshakeEx
 			generateKeys(premasterSecret);
 			break;
 		case PSK:
+			recordLayer.pauseRetransmission();
```

Comment:
But I agree, that just stop the retransmission on an incoming handshake message would be a very simple solution for "longer processing times" and I don't see a easy argument against it. But not being an security expert, the unwanted side effects are out of my scope.